### PR TITLE
Linking Docker image source

### LINKED ISSUE #1118: Docs request: linking Docker container
https://github.com/go-skynet/LocalAI/blob/v1.30.0/docker-compose.yaml#L5

```
image: quay.io/go-skynet/local-ai:latest
```

Where is this container hosted, is it Docker Hub?  I would like to add a link in the docs for it, just need to know where it resides

### PR DESCRIPTION
**Pull request by jamesbraza**
Closes https://github.com/go-skynet/LocalAI/issues/1118

### CODE REVIEW

File: content/howtos/easy-model-import-gallery.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ weight = 2
 
 Now lets pick a model to download and test out. We are going to use `luna-ai-llama2-uncensored.ggmlv3.q5_K_M.bin`, there are a few ways to do this, https://huggingface.co/TheBloke/Luna-AI-Llama2-Uncensored-GGML/blob/main/luna-ai-llama2-uncensored.ggmlv3.q5_K_M.bin
 
-In the `docker` cmd run this. This uses the Gallery to download the model, it may also set up the yaml file, but we will need to override that for the how to setup!
```
Comment (lunamidori5) [inline content/howtos/easy-model-import-gallery.md:10]: This will need to be added back in. As there was some confusion on where to run the command.

**Review by lunamidori5 — CHANGES_REQUESTED** [review]
The changes to import-gallery will need reworded / added. That bit was added for clearness of where to run the command

**Comment by lunamidori5** [comment]
(Even tho, yes it could be ran where ever you want, its easy to just tell you to run it in the docker itself... If the user knows they know)

File: content/howtos/easy-model-import-gallery.md (hunk 1 above)
Comment (jamesbraza) [reply content/howtos/easy-model-import-gallery.md:10]: Okay I see what you mean now.  I have added it back in, but expanded upon it to make more sense.

Let me know what you think!

**Review by lunamidori5 — APPROVED** [review]

**Pull request merged**